src/monitor/pgautofailover.sql

```diff
@@ -40,7 +40,8 @@ CREATE TYPE pgautofailover.replication_state
     'wait_maintenance',
     'report_lsn',
     'fast_forward',
-    'join_secondary'
+    'join_secondary',
+    'dropped'
```

Comment:
This new `dropped` one is not added in the migration.

Reply:
Good catch, fixed now!